### ISSUE TITLE
during abort_repair failed with NameError: global name 'UnexpectedExit' is not defined

### ISSUE THREAD

**Issue by fgelcer** [scylladb/scylla-cluster-tests#1155]
### Prerequisites

* [ ] Are you rebased to master ? - **no, 3.1**
* [ ] Is it reproducible ?
* [x] Did you perform a cursory search if this issue isn't opened ?

### Versions
* **SCT**: [branch or hash tag you are working on] 3.1
* **scylla**: [branch or commit hash] Tomek rpm - scylla-server-3.1.0.rc2-0.20190618.fa53994fb0.el7.x86_64.rpm

### Logs
* test_id: 40f51427-00b8-4ece-89e1-0263636b1c1e
* [job log:](https://cloudius-jenkins-test.s3.amazonaws.com/40f51427-00b8-4ece-89e1-0263636b1c1e/job-2019-07-14T14.05-cbec098.zip)

### Description

nemesis abort_repair caused exception:
```
L0121 INFO | RemoteCmdRunner [centos@172.30.0.88]: Command "nodetool -h localhost repair" finished with status 2
L0642 DEBUG| Exception in thread Thread-55353:
L0642 DEBUG| Traceback (most recent call last):
L0642 DEBUG|   File "/usr/lib64/python2.7/threading.py", line 812, in __bootstrap_inner
L0642 DEBUG|     self.run()
L0642 DEBUG|   File "/usr/lib64/python2.7/threading.py", line 765, in run
L0642 DEBUG|     self.__target(*self.__args, **self.__kwargs)
L0642 DEBUG|   File "/sct/sdcm/nemesis.py", line 552, in repair_nodetool_repair
L0642 DEBUG|     self._run_nodetool(repair_cmd, node)
L0642 DEBUG|   File "/sct/sdcm/nemesis.py", line 163, in _run_nodetool
L0642 DEBUG|     except (UnexpectedExit, Failure) as details:
L0642 DEBUG| NameError: global name 'UnexpectedExit' is not defined
L0642 DEBUG| 
L0852 DEBUG| sdcm.nemesis.ChaosMonkey: Execute a complete repair for target node
L0221 DEBUG| RemoteCmdRunner [centos@172.30.0.88]: Running command "nodetool -h localhost repair"...
```

**Comment by fgelcer** [comment]
[jenkins job](http://jenkins.scylladb.com/view/staging/job/scylla-staging/job/fabio/job/3h_reproducer_malformed_sstables_tomek_version/2/)

**Comment by fgelcer** [comment]
i had another UnexpectedExit exception, now with a different error and on a different test.
this happened during nemesis show_toppartitions

```
2019-07-14 23:27:35,978 nemesis          L0540 ERROR| sdcm.nemesis.ChaosMonkey: Exception in random_disrupt_method show_toppartitions: Encountered a bad command exit code!

Command: 'cqlsh --request-timeout=300 --no-color --execute "SELECT keyspace_name, table_name from system_schema.tables" Node longevity-10gb-3h-jenkins-db-node-33eaa49a-1 [52.23.232.3 | 172.30.0.244] (seed: True)'

Exit code: 1

Stdout:



Stderr:

bash: -c: line 0: syntax error near unexpected token `seed:'
bash: -c: line 0: `cqlsh --request-timeout=300 --no-color --execute "SELECT keyspace_name, table_name from system_schema.tables" Node longevity-10gb-3h-jenkins-db-node-33eaa49a-1 [52.23.232.3 | 172.30.0.244] (seed: True)'


2019-07-14 23:27:35,978 nemesis          L1041 ERROR| sdcm.nemesis.ChaosMonkey: Unhandled exception in method <function disrupt at 0x7fa0bf016140>
Traceback (most recent call last):
  File "/sct/sdcm/nemesis.py", line 1037, in wrapper
    result = method(*args, **kwargs)
  File "/sct/sdcm/nemesis.py", line 1249, in disrupt
    self.call_random_disrupt_method()
  File "/sct/sdcm/nemesis.py", line 538, in call_random_disrupt_method
    disrupt_method()
  File "/sct/sdcm/nemesis.py", line 969, in disrupt_show_toppartitions
    args = generate_random_parameters_values()
  File "/sct/sdcm/nemesis.py", line 954, in generate_random_parameters_values
    ks_cf_list = self.loaders.get_non_system_ks_cf_list(self.loaders.nodes[0], self.cluster.nodes[0])
  File "/sct/sdcm/cluster.py", line 3031, in get_non_system_ks_cf_list
    .format(request_timeout, cmd, db_node_ip), verbose=False)
  File "/sct/sdcm/remote.py", line 230, in run
    watchers=watchers, command_timeout=timeout)
  File "<decorator-gen-3>", line 2, in run
  File "/usr/lib/python2.7/site-packages/fabric/connection.py", line 30, in opens
    return method(self, *args, **kwargs)
  File "/usr/lib/python2.7/site-packages/fabric/connection.py", line 721, in run
    return self._run(self._remote_runner(), command, **kwargs)
  File "/usr/lib/python2.7/site-packages/invoke/context.py", line 101, in _run
    return runner.run(command, **kwargs)
  File "/usr/lib/python2.7/site-packages/invoke/runners.py", line 283, in run
    return self._run_body(command, **kwargs)
  File "/usr/lib/python2.7/site-packages/invoke/runners.py", line 431, in _run_body
    raise UnexpectedExit(result)
UnexpectedExit: Encountered a bad command exit code!

Command: 'cqlsh --request-timeout=300 --no-color --execute "SELECT keyspace_name, table_name from system_schema.tables" Node longevity-10gb-3h-jenkins-db-node-33eaa49a-1 [52.23.232.3 | 172.30.0.244] (seed: True)'

Exit code: 1

Stdout:



Stderr:

bash: -c: line 0: syntax error near unexpected token `seed:'
bash: -c: line 0: `cqlsh --request-timeout=300 --no-color --execute "SELECT keyspace_name, table_name from system_schema.tables" Node longevity-10gb-3h-jenkins-db-node-33eaa49a-1 [52.23.232.3 | 172.30.0.244] (seed: True)'
```

**Comment by fruch** [comment]
@fgelcer that's two different issues, can you open a new one for the one you just found ?

**Comment by bentsi** [comment]
it is an issue that was resolved in master and 3.1-staging
